docs/source/workflows/functions/text-to-sql.md

```
@@ -0,0 +1,521 @@
+- `<db_host>`: Your Databricks workspace URL, for example `your-workspace.cloud.databricks.com`
+- `<http_path>`: Path to your SQL warehouse or compute cluster, for example `/sql/1.0/warehouses/abc123`
+- `<catalog>`: Catalog name, for example `main`
+- `<schema>`: Schema name, for example `default`
+
+**Example:**
+```bash
+CONNECTION_URL="databricks://token:dapi-xxx@your-workspace.cloud.databricks.com:443/default?http_path=/sql/1.0/warehouses/abc123&catalog=main&schema=default"
+```
+
+**Note**: Only Databricks is currently supported. The connection uses SQLAlchemy with the `databricks-sql-connector` driver. Other databases can be customized as following:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)